packages/python-galaxy-importer/python-galaxy-importer.spec

```diff
@@ -26,24 +26,24 @@ Requires:       /usr/bin/ansible-test
 %if 0%{?rhel} == 8
 # We only have ansible-lint built on EL8
 Requires:       ansible-lint < 6.0
-Requires:       ansible-lint >= 5.0.0
+Requires:       ansible-lint >= 5.0.8
 %endif
-Requires:       python%{python3_pkgversion}-attrs < 21
-Requires:       python%{python3_pkgversion}-attrs >= 19.3.0
+Requires:       python%{python3_pkgversion}-attrs < 22
+Requires:       python%{python3_pkgversion}-attrs >= 21.2.0
 Requires:       python%{python3_pkgversion}-bleach < 4
 Requires:       python%{python3_pkgversion}-bleach >= 3.3.0
 Requires:       python%{python3_pkgversion}-bleach-allowlist < 2
 Requires:       python%{python3_pkgversion}-bleach-allowlist >= 1.0.3
 Requires:       python%{python3_pkgversion}-flake8 < 4
-Requires:       python%{python3_pkgversion}-flake8 >= 3.7.9
+Requires:       python%{python3_pkgversion}-flake8 >= 3.9.2
```

Comment:
This really looks like a development dependency, but given it was already there ...

Reply:
No, it is not. It's running flake on the python code in the ansible collections it imports.